packages/utils/interactive-hooks/src/Pressability/Pressability.ts

```diff
@@ -395,7 +395,8 @@ export class Pressability {
         ? null
         : {
             onMouseEnter: (event: MouseEvent): void => {
-              if (isHoverEnabled()) {
+              const { disabled } = this._config;
+              if (!disabled && isHoverEnabled()) {
```

Comment:
@SaadNajmi this is a short term workaround so that Krystal can make progress on her changes. I have an item logged to make Loading button not change visuals on hover and remove this, so no worries about bringing it upstream.